Mathlib/GroupTheory/Exponent.lean

```diff
@@ -286,6 +286,59 @@ theorem _root_.Nat.Prime.exists_orderOf_eq_pow_factorization_exponent {p : ℕ}
 #align nat.prime.exists_order_of_eq_pow_factorization_exponent Nat.Prime.exists_orderOf_eq_pow_factorization_exponent
 #align nat.prime.exists_order_of_eq_pow_padic_val_nat_add_exponent Nat.Prime.exists_addOrderOf_eq_pow_padic_val_nat_add_exponent
 
+open Nat in
+lemma _root_.Nat.coprime_factorization_lcm_left_factorization_lcm_right (a b : Nat) :
```

Comment:
I would also put this into `[Mathlib/Data/Nat/Factorization/Basic.lean](https://github.com/leanprover-community/mathlib4/pull/11235/files#diff-5a35c7fed7b39fd34329e2d6cea2700838ca70eb25ce505a38c289e0b093a94d)` (unless it needs some import that is missing there) since it is a relevant property of `factorization_lcm_left` and `factorization_lcm_right` and not specific to group exponents.

Reply:
Ah yes, I forgot to mention it: it needs new imports. But I realize that `Factorization.Basic` already has pretty heavy imports, maybe it's not really a problem. Let me try.

Reply:
I've moved it, adding the import. I think it is ok.